### DISCUSSION TITLE
Seems the project.godot file contains dalogic data

### DISCUSSION THREAD

**Discussion by jaykop** [dialogic-godot/dialogic#2386] [Q&A]
When I added a sample timeline and variable, it changes the project.godot file like this;

[dialogic]

directories/dch_directory={}
directories/dtl_directory={
"SampleTimeline": "res://Data/Dialogic/Timeline/SampleTimeline.dtl"
}
variables={
"Choice1": "Ch1",
"Choice2": "Ch2"
}

is this ok for the long-term game dev?
I mean, probably my game dialogue data will be huge and it would make the size of project.godot file bigger.
I assume that is not quite ideal.
Is there any setting that I missed for this? or is this just how Dialogic works?

**Comment by Jowan-Spooner** [comment]
This is just how dialogic works. 

If you could explain how this would become a problem, please let us know. Sure the project.godot file get's bigger but otherwise we would have to store these things in a different file.

**Reply by jaykop** [reply]
Thank you for the answer!

Well, I was just wondering.
The project file usually contains project configs or settings but this seems more like data so I thought "is saving data here ok?"
Asked chatgpt and it says; "bigger project file would make project slower" so I got worried cuz this "data" gonna bigger.

If this is not a problem and how dialogic works, I keep using this with relief.

Thanks again!
